docs/api/crash-reporter.md

```diff
@@ -27,6 +27,10 @@ crashReporter.start({
     * Only string properties are send correctly.
     * Nested objects are not supported.
 
+**Note:** On OS X, electron uses a new `crashpad` client, which is different
+with the `breakpad` on Windows and Linux. To enable crash collection feature,
+you are required to call `crashReporter.start` API to initiliaze `crashpad` in Browser Process, even you only collect crash report in Renderer Process.
```

Comment:
We can add a new hidden API to initialize crashpad and call it automatically in `crashReporter.start`, and in renderer process the call to initialize crashpad can just be forwarded to the browser process, so we don't require users to do this.